gomock_reflect_2829919009/prog.go

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
This file was probably checked in by mistake

Reply:
Deleted.